src/OpenTelemetry.Extensions.Hosting/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Removed the `OpenTelemetryBuilder.StartWithHost` extension and moved the
```

Comment:
Lets make it here itself that, with this change, this whole package itself will not be required anymore, and will be deprecated with no further release.

Reply:
+1